media-gfx/maim/maim-5.7.4.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Always aim to use the latest available EAPI. In this case, 8.

Reply:
Thank you! This was one of my first ebuilds. Now I know a lot more and your comments are amazingly helpful. I will keep all of them in mind for the future.